src/Select.soy

```diff
@@ -39,7 +39,7 @@
 			{param expanded: $disabled ? false : $expanded_ /}
 			{param header kind="html"}
 				<button
-					class="{$buttonClass ? $buttonClass : ''} dropdown-select"
+					class="{$buttonClass ?: 'btn btn-default'} dropdown-select"
```

Comment:
For user experience, I think it's always good to have the component rendered without blinking, even though we have to handle with those CSS classes in two places.